openedx/core/djangoapps/content_libraries/docs/decisions/0003-course-content-independence.rst

```diff
@@ -0,0 +1,41 @@
+3. Course Content should not depend on Library Content
+------------------------------------------------------
+
+Status
+------
+
+Accepted
+
+
+Context
+-------
+
+When a course uses content from a library via a ``LibraryContentBlock``, it creates a course-local copy of that content. It will also store certain metadata in the ModuleStore that connect the course copy with the original in the library, such as the usage key and version of the original, as well as the library-suggested values for various fields (e.g. ``num_attempts=10``).
```

Comment:
```suggestion
When a course uses content from a library via a ``LibraryContentBlock``, it creates a course-local copy of the library's blocks. For each library block, it will also store certain metadata in the ModuleStore that connects the course copy with the original in the library, such as the usage key and version of the original, as well as the library-suggested values for various fields (e.g. ``num_attempts=10``).
```

on my first read-through I took "content" to mean "a set of blocks" rather than "each block". this might clarify it a bit.